bbb-common-web/src/main/java/org/bigbluebutton/api/ParamsProcessorUtil.java

```diff
@@ -29,6 +29,10 @@
 import java.util.*;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import com.google.gson.Gson;

```

Comment:
In other spots in akka-apps we've used `spray.json`. Have you considered using it or is there a specific `gson` feature you like?

Reply:
Spray.json seems to be a library to Scala.
And this paramProcessor is coded in Java, so I used Gson that is already imported to the project. Moreover, Gson is being widely used in the project to convert log objects to Json.